### ISSUE TITLE
fs.watch does not work on Ubuntu 

### ISSUE THREAD

**Issue by 1a1a11a** [mateogianolio/sshync#28]
This is a great tool, but I just notice that it does not work on Ubuntu because fs.watch is not available. It would be nice to explicitly on the README.

**Comment by mateogianolio** [comment]
True, on linux the `recursive` option does not work (see https://nodejs.org/api/fs.html#fs_caveats). There are other ways to watch recursively, the most viable option is probably [chokidar](https://github.com/paulmillr/chokidar). I would accept a PR that replaces `fs.watch`.

**Comment by 1a1a11a** [comment]
Thank you! I am not a node user, but sshync is easy to understand so I will try to create a PR soon.
